test/functional/rpc_rawtransaction.py

```diff
@@ -42,7 +42,6 @@ class RawTransactionsTest(BitcoinTestFramework):
     def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 3
-        # TODO: remove -txindex. Currently required for getrawtransaction call.
```

Comment:
I removed this todo from the code, but its still on my personal list :) I am interested in improving this test so it actually covers the unique intended functionalities. Eg. since we are currently passing  `-txindex` to every node, this test wouldn’t catch if `getrawtransaction` using just the blockhash started failing. I’ve started to dive in and I’m headed down the path of using `-txindex`  on some (not all) of the nodes to make the tests more robust. Does this seem reasonable?